patches/common/boringssl/.patches

```diff
@@ -1,3 +1,4 @@
+compatibility_evp_ciph_ocb_mode.patch
```

Comment:
Was this patch added manually?  I'm not sure how this could have been added to the top of the file if the `export-patches` script was used

Reply:
This patch was added manually. Shall I re-add it with the `export-patches` command?

Reply:
> Was this patch added manually? I'm not sure how this could have been added to the top of the file if the `export-patches` script was used

This patch was added manually. Shall I re-add it with the export-patches command?

Reply:
@TaikiAkita Yep, the flow for making these patches is like this

* Run `gclient sync --your --flags --here`
* Add your commit on top of the checked out `boringssl`
* Run `git-export-patches -o electron/patches/common/boringssl`

👍

Reply:
@MarshallOfSound 

I regenerated the patches with `git-export-patches`, please have a look.